### PR TITLE
#158626101 Clear Cache after Deleted Muscle(s)

### PR DESCRIPTION
**Pull request by tonykimathi**
#### What does this PR do?
Reset the cache after a muscle has been deleted.
#### Description of Task to be completed?
It seems that after deleting a muscle, it keeps appearing in descriptions etc. This is probably due to the cache not being correctly reset.

The cache should be reset after a muscle has been deleted
#### What are the relevant pivotal tracker stories?
[#158626101](https://www.pivotaltracker.com/story/show/158626101)
#### Screenshots (if appropriate)

> Exercise Template before Resetting Cache

image

> Exercise Template after Resetting Cache

image

#### Questions:

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 }
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = ')8-woct4l-fk2(gs&z@uyizjnaw_yg#5rm8$m_xx8szpb)lyxp'
```
Comment (parseen254) [inline settings.py:28]: @tonykimathi this is a security disaster waiting to happen.

File: wger/exercises/signals.py [hunk 2]
```diff
@@ -15,13 +15,23 @@
...
+from wger.utils.cache import cache
+
+
+@receiver(pre_delete, sender=Muscle)
+def clear_muscle_cash_on_delete(sender, instance, **kwargs):
+    '''
+    Clears the cache after muscle is deleted
+    '''
+
+    cache.clear()
```
Comment (parseen254) [inline wger/exercises/signals.py]: @tonykimathi Why clear the whole cache not only muscle related cache?

**Review by parseen254 — CHANGES_REQUESTED** [review]

**Comment by parseen254** [comment]
Also @tonykimathi rebase with develop

File: settings.py (hunk 1 above)
Comment (tonykimathi) [reply settings.py:28]: 🤦‍♂️ How did I not notice. Thanks. Fixed that.

File: wger/exercises/signals.py (hunk 2 above)
Comment (tonykimathi) [reply wger/exercises/signals.py]: @parseendavid Yes. Clearing only muscle related cache is more efficient. Kindly check my updated code.

**Review by parseen254 — APPROVED** [review]

**Pull request merged**